### ISSUE TITLE
Typescript error Missing types for "js-cookie" after updating to latest version from 13.14.3

### ISSUE THREAD

**Issue by JustFly1984** [streamich/react-use#892]
Missing types "@types/js-cookie": "^2.2.4"

After installing types, it errors out with incorrect import/export.

**Comment by JustFly1984** [comment]
reverting back to 13.14.3 solves issue

**Comment by ayush987goyal** [comment]
@streamich SI think this might be the issue with the `useCookie` hook. Should we move the `@types/js-cookie` from `devDependencies` to `dependencies`? Or should we re-export the single import of `CookieAttributes` from the package itself?

**Comment by ayush987goyal** [comment]
@streamich Fixed with #894 and in `v13.16.1`. Can be closed.

**Comment by JustFly1984** [comment]
Still having same issue with 13.16.1:
<img width="454" alt="Screenshot 2020-01-15 04 42 30" src="https://user-images.githubusercontent.com/2255993/72397723-b9f5e500-3751-11ea-9924-d6f64c383936.png">

**Comment by JustFly1984** [comment]
please reopen

**Comment by ayush987goyal** [comment]
Can you please set `esModuleInterop` flag to `true` in your `tsconfig.json` and try once?

**Comment by JustFly1984** [comment]
setting up `esModuleInterop` itself to true is not enough, it also requires to set `"allowSyntheticDefaultImports": true`, to get rid of an error in react-use, but that causes a lot of other dependencies issues. I'm reverting it back.

**Comment by ayush987goyal** [comment]
Just tested `v13.17.0` with scaffolding a new app with `create-react-app` `TypeScript` template and setting both `esModuleInterop` and `allowSyntheticDefaultImports` to `false` (it scaffolds with both set to `true`). It works without any issues. Maybe something else is not correct in your environment.

**Comment by ayush987goyal** [comment]
@streamich We *can* make the `js-cookie` import to 
```ts
import * as Cookies from 'js-cookie';
```
But I am not sure if we *should* do that

**Comment by streamich** [comment]
@ayush987goyal I don't think so, it should not matter how we internally import it, right?

**Comment by ayush987goyal** [comment]
Yes that is correct. Therefore I gave it a try in a TS react project as mentioned above and it worked just fine.

**Comment by JustFly1984** [comment]
I still have same issue with 13.18.0

<img width="802" alt="Screenshot 2020-01-16 12 17 23" src="https://user-images.githubusercontent.com/2255993/72511033-80110580-385b-11ea-9909-26f6324259eb.png">

**Comment by JustFly1984** [comment]
I guess issues could be with @types/js-cookies itself

**Comment by JustFly1984** [comment]
Still having same issue in 13.20.0. Had cleaned ./cache, node_modules and yarn.lock file, still have same issue.

PS project is based on Gatsby.js. I'll try to make minimal reproduction on codesandbox.io

**Comment by makanti** [comment]
> @streamich Fixed with #894 and in `v13.16.1`. Can be closed.

Hello there, this issue has come back. It looks like the fix on #894 in `v13.16.1` was reverted since then. It moved from `Dependencies` back to `DevDependencies` (Check here: https://github.com/streamich/react-use/compare/v15.3.8...v17.0.2). I am trying to upgrade from `v15.3.8` to `v17.0.1` and this breaks the types. Any idea why has `@types/js-cookie` moved back to `DevDependencies`? Thanks! :)

![image](https://user-images.githubusercontent.com/5037924/106501951-a7189480-64bb-11eb-9543-91d38a7633bf.png)

**Comment by Paden** [comment]
I am experiencing the same problem as @makanti. Any updates?

**Comment by Hernanarica** [comment]
Pude encontrar la solucion para TypeScript, hay que instalar npm i js-cookie y npm i @types/js-cookie -D como devDependencie y tener ambos instalados
